### PR TITLE
Fixed broken roadmap links

### PR DESCRIPTION
**Pull request by tonkatsu7**

### CODE REVIEW

File: _posts/2017-10-21-roadmap.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ title: In Progress
 
 **High-level features we are working on and features we have committed to implementing**
 
-> See our [roadmap on Github](https://github.com/orgs/SoftwareEngineeringDaily/projects/1). If you have an idea for a feature, [email us](mailto:ideas@softwareengineeringdaily.com) or [join Slack](http://softwaredaily.herokuapp.com/)!
```
Comment (crablar) [inline _posts/2017-10-21-roadmap.md]: This one is actually correct as is--the brainstorming project board is another representation of the roadmap on Github.

File: vision.html [hunk 2]
```diff
@@ -10,5 +10,5 @@
   Users can discuss content, share links, and connect with each other. As you use Software Engineering Daily, the application figures out your tastes, and recommends people for you to connect with and content that you might find entertaining or educational.
   <br>
   <br>
-  To learn more, check out <a href="http://localhost:4000/roadmap/">our roadmap.</a>
```
Comment (crablar) [inline vision.html:13]: Good catch, this one is definitely wrong!

File: _posts/2017-10-21-roadmap.md [hunk 3]
```diff
@@ -77,4 +77,4 @@ We need to build a recommendation system that can suggest podcasts, content, oth
 
 ![Pipeline](https://user-images.githubusercontent.com/1283090/32086696-d0c05abc-ba8b-11e7-8ab5-88564d4feb31.png)
 
-> See our [roadmap on Github](https://github.com/orgs/SoftwareEngineeringDaily/projects/1). If you have an idea for a feature, [email us](mailto:ideas@softwareengineeringdaily.com) or [join Slack](http://softwaredaily.herokuapp.com/)!
```
Comment (crablar) [inline _posts/2017-10-21-roadmap.md]: This one is also OK!

**Review by crablar — COMMENTED** [review]
2 of your changes are good corrections, 2 of them are actually correct links (though I understand why they are confusing...).

File: _posts/2017-10-21-roadmap.md (hunk 1 above)
Comment (tonkatsu7) [reply _posts/2017-10-21-roadmap.md]: Ah no worries then

**Comment by tonkatsu7** [comment]
Reverted the 2 good links on the roadmap post where the brainstorming project board is another representation of the roadmap on Github

**Comment by crablar** [comment]
@tonkatsu7 thanks for those fixes--I actually should have linked to https://github.com/SoftwareEngineeringDaily/softwareengineeringdaily.github.io/projects/1

**Pull request merged**